assets/shaders/octree/_coneTrace.glsl

```diff
@@ -76,20 +77,19 @@ vec4 coneTrace(
     int steps = 0;
     
     // Move the cone origin so it doesn't intersect with own voxels
-    vec3 offsetedConeOrigin = coneOrigin + coneDirection * voxelSize * 2;
+    vec3 offsetedConeOrigin = coneOrigin + coneDirection * voxelSize * 1.4;
     while (distanceAlongCone < maxDistance && returnColor.a < 0.97) {
         float coneDiameter = clamp(coneDiameterCoefficient * distanceAlongCone, 0.0009765625, 100.0);
         float lod = calculateLod(coneDiameter);
         uint octreeLevel = uint(ceil(lod));
         float parentWeight = octreeLevel - lod; // Non-linear, we should approximate the log with many lines
 
         bool changedOctreeLevel = octreeLevel != previousOctreeLevel;
-        if (changedOctreeLevel) {
-            // To account for the larger voxelSize in the new level
-            sampleStep *= 2; // Increase sampleStep, same as increasing voxelSize by 2
-        }
 
         vec3 queryCoordinates = offsetedConeOrigin + distanceAlongCone * coneDirection;
+        if(isOutsideRange(queryCoordinates, vec3(0), vec3(1))) {
+          break;
+        }
```

Comment:
This is not a good solution, but doing nothing was worse. Should probably skip the iteration as if it didn't find a node with traverse octree if it is outside this range